exercises/practice/palindrome-products/.articles/is-palindrome-check/content.md

```diff
@@ -0,0 +1,165 @@
+# Is number a palindrome? 
+
+There are multiple approaches to solve the palindrome product exercise. 
+Most of them require us to check if a given number is a palindrome. 
+
+## Approaches
+There are multiple ways to do it in C# and we will look at them in this article. 
+
+### String reversal
+
+Possibly the most obvious solution is to treat a number as a string of characters
+and reverse it. Let's have a look:
```

Comment:
You're currently breaking up lines at a cutoff point, but our [style guide](https://exercism.org/docs/building/markdown/markdown#h-one-sentence-per-line) suggests using one sentence per line (no cutoff). That would make this:

```suggestion
Possibly the most obvious solution is to treat a number as a string of characters and reverse it.
Let's have a look:
```

Of course, this would also apply elsewhere.

Reply:
And maybe be explicit about the last part of the logic too (comparing the strings):

```suggestion
The most straightforward solution is to treat a number as a string of characters and see if it is equal to the reversed string.
Let's have a look:
```

Feel free to change the wording.